XenAdmin/Wizards/PatchingWizard/PatchingWizard_SelectPatchPage.cs

```diff
@@ -380,6 +380,7 @@ private void BrowseButton_Click(object sender, EventArgs e)
                             if (!string.IsNullOrEmpty(unzippedFilePath))
                             {
                                 unzippedUpdateFilePath = unzippedFilePath;
+                                PatchingWizard.PatchesToRemove.Add(unzippedUpdateFilePath);

```

Comment:
I suggest having the list of unzipped files as a property of this page and then the wizard can simply add this list to its own list of files that will eventually be cleaned up.

In general, in the wizards we do things this way: 
- The page doesn't know anything about the wizard (so it can be added to any wizard); 
- The page publishes some properties, which the wizard can access (because the wizard knows about its pages).